src/App.tsx

```diff
@@ -1,10 +1,45 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Filter } from './types/Filter';
+import { Error } from './types/Error';
 
-const USER_ID = 0;
+const USER_ID = 6335;
 
 export const App: React.FC = () => {
+  const [allTodos, setTodos] = useState<Todo[]>([]);
+  const [isError, setIsError] = useState<Error | null>(null);
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+  const [isErrorHidden, setIsErrorHidden] = useState(true);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        setIsError(Error.Load);
+        setIsErrorHidden(false);
+        setTimeout(() => setIsErrorHidden(true), 3000);
+      });
+  }, []);
+
+  const filterHandler = (array: Todo[], filterType: string) => {
+    switch (filterType) {
+      case Filter.Active:
+        return array.filter(item => !item.completed);
+      case Filter.Completed:
+        return array.filter(item => item.completed);
+      default:
+        return array;
+    }
+  };
+
+  const activeTodos = allTodos.filter(todo => !todo.completed);
```

Comment:
why don't use filterHandler  as well?
```suggestion
  const activeTodos = filterHandler(allTodos, Filter.Active);
```